basis/io/sockets/secure/secure.factor

```diff
@@ -44,16 +44,20 @@ HOOK: <secure-context> secure-socket-backend ( config -- context )
         with-disposal
     ] with-scope ; inline
 
-TUPLE: secure { addrspec read-only } ;
+TUPLE: secure
+    { addrspec read-only }
+    { hostname read-only } ;
 
-C: <secure> secure
+: <secure> ( addrspec host -- secure )
```

Comment:
It's a leftover from when I (temporarily) introduced a `<secure-host>` variant to aid in transitioning over to requiring the hostname everywhere. Fixed.